Transactions.md

```diff
@@ -169,3 +169,45 @@ Where
 
 Where
   - `P2WPKH(Local Penalty Public Key)`'s value is `P2WSH(Remote's ToLocalOutput).value - fee`
+
+## Mutual Closing Transaction
+### <a name="MutualClosingKnownValues">Known Values</a>
+  * Local Mutual Closing Public Key: `ECPublicKey`
+  * Local Payout: `CurrencyUnit`
+  * Remote Mutual Closing Public Key: `ECPublicKey`
+  * Remote Payout: `CurrencyUnit`
+  * nLockTime: `UInt32`
+  * DLC Funding Output: `ScriptPubKey`
+  * Fee Rate: `FeeUnit`
+  
+Where
+  - After the contract maturity time, Mutual Closing Transaction is made for fee reduction 
```

Comment:
should probably also mention improved privacy, since not using Mutual Closing Tx means your txs have a very distinct template

Reply:
Added about development in privacy.